crates/biome_formatter/src/lib.rs

```diff
@@ -196,15 +197,19 @@ impl std::fmt::Display for LineEnding {
     }
 }
 
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, Merge, Hash, PartialEq)]
 #[cfg_attr(
     feature = "serde",
-    derive(serde::Serialize, serde::Deserialize, schemars::JsonSchema),
+    derive(serde::Serialize, schemars::JsonSchema),
     serde(rename_all = "camelCase")
 )]
 pub struct IndentWidth(u8);
 
 impl IndentWidth {
+    pub const MIN: u8 = 2;
+
+    pub const MAX: u8 = 80;
```

Comment:
Why `80`? Could `8` be a better upper limit?

Reply:
There is no particular reason, but `8` seems too low. Maybe `20`?

Reply:
Let's take 16 or 24.

Reply:
Done